frontend/src/services/http/http.service.ts

```diff
@@ -4,7 +4,10 @@ import { ContentType, HttpHeader, HttpMethod } from 'common/enums';
 import { HttpOptions } from 'common/types';
 
 class Http {
-  load<T = unknown>(url: string, options: Partial<HttpOptions> = {}): Promise<T> {
```

Comment:
It would be better to add `hasAuth` flag with default `true` value to the `HttpOptions`

And then, add token, only by this flag